Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -425,6 +433,7 @@ export default class TimelineView extends Vue {
                         );
                     }
                     this.sortEntries();
+                    this.noteCount = results.resourcePayload.length;
```

Comment:
does this get updated when a new note is added or deleted?

Reply:
good point. can add a watcher

Reply:
done